kairon/shared/auth.py

```
@@ -180,6 +180,25 @@ def create_store_page_token(*, data: dict, token_type: TOKEN_TYPE = TOKEN_TYPE.S
+    ):
+        from kairon.shared.data.constant import TOKEN_TYPE
+        raw_token = (request.headers.get("authorization") or token or "")
+        if raw_token.lower().startswith("bearer "):
+            raw_token = raw_token[7:]
+        if raw_token:
+            try:
+                claims = Utility.decode_limited_access_token(raw_token)
+                if claims.get("type") == TOKEN_TYPE.STORE_PAGE.value:
+                    return Authentication.validate_store_page_token(request)
+            except Exception:
+                pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)